### PR TITLE
chore: notify me when users encounter a bug

### PR DESCRIPTION
**Pull request by kormide**
Addresses https://github.com/bazel-contrib/publish-to-bcr/issues/38.

### CODE REVIEW

File: deployment/environments/dev/variables.tf [hunk 1]
```diff
@@ -34,6 +34,12 @@ variable "notifications_email" {
     default = "no-reply@aspect.dev"
 }
 
+variable "debug_email" {
+    type = string
+    description = "Email address to notify development team of issues"
+    default = "derek@aspect.dev"
```
Comment (alexeagle) [inline deployment/environments/dev/variables.tf:40]: nit: maybe we should create a Google Group so you're not the only one on the distribution list?

**Review by alexeagle — APPROVED** [review]
okay with me to land as-is

File: deployment/environments/dev/variables.tf (hunk 1 above)
Comment (kormide) [reply deployment/environments/dev/variables.tf:40]: Yeah that would be nice. Can you set up `publish-to-bcr-dev@aspect.com` or something of the sort?

Comment (kormide) [reply deployment/environments/dev/variables.tf:40]: Oh, you meant a Google group, not a mailing group? Though maybe those aren't so different. I'll look into it.

**Pull request merged**